### DISCUSSION TITLE
Can Not Import xlsx.table from janitor

### DISCUSSION THREAD

**Discussion by DanielPerkins472** [pyjanitor-devs/pyjanitor#1145] [Q&A] [answered]
Hello! Im trying to use your xlsx_table function from janitor to accomplish the above, however Im unable to import the xlsx_table function. I have successfully downloaded / installed pyjanitor and import the package into the jupyter notebook, however I can failures when I try to import xlsx_table from janitor. Do you have any advice?

![image](https://user-images.githubusercontent.com/111129483/184356630-8c499259-8360-429d-bded-f35d275275ba.png)

**Comment by Zeroto521** [comment, marked as answer]
Sorry for being late.

Hi, I saw your pyjanitor version is 0.22.0.
pyjanitor 0.22 doesn't have `xlsx_table`, this function was added into pyjanitor at 0.23 #990.
Now you could easily update pyjanitor and use it.

You may mislead by the docs website, the website always points to the latest version, not the stable version.
